sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -99,22 +98,12 @@ fun MapOptions.Builder.applyDefaultParams(context: Context): MapOptions.Builder
 fun ResourceOptions.Builder.applyDefaultParams(
   context: Context
 ): ResourceOptions.Builder = also {
-  // make sure that directory `/mapbox/maps` exists
-  val databaseDirectoryPath = "${context.filesDir.absolutePath}/$DATABASE_PATH"
-  val databaseDirectory = File(databaseDirectoryPath)
-  if (!databaseDirectory.exists()) {
-    if (!databaseDirectory.mkdirs()) {
-      throw IllegalStateException("Unable to create database folder")
-    }
-  }
-
   // check in the resources
   val tokenResId = ResourceOptionsManager.getTokenResId(context)
   // Apply the token from resources.
   if (tokenResId != 0) {
     accessToken(context.getString(tokenResId))
   }
 
-  cachePath("$databaseDirectoryPath/$DATABASE_NAME")
-  cacheSize(DEFAULT_CACHE_SIZE) // 50 mb
+  dataPath("${context.filesDir.absolutePath}/$DATA_PATH")
```

Comment:
datapath = ".mapbox/maps/", is it expected that gl-native will provide the actual database name?

Reply:
Yes, it will be named as map_data.db, and actually we don't even have to specify the dataPath, gl-native will use the default for that tilestore is using(.mapbox)